recipe/conda_build_config.yaml

```diff
@@ -2,6 +2,6 @@ opencl_impl:  # [osx]
 - apple       # [osx]
 - khronos     # [osx]
 channel_targets:
-- conda-forge main
+- conda-forge openmm_rc
```

Comment:
Yea it looks like this sneaked in. Gotta revert and rerender in a new PR.

Reply:
I'll take a stab at this

Reply:
Oops!  Thanks for catching that!  And thanks Matt, that would be great.